src/unity/paths/operator_accounts_accountId.yml

```diff
@@ -24,7 +24,7 @@ get:
                   organizations:
                     type: array
                     description: organizations in the account
-                    $ref: '../schemas/Organizations.yml'
+                    $ref: '../schemas/OperatorOrganizations.yml'
```

Comment:
This and the other changes like it will be a breaking change in the UI, so we'll need to update the UI immediately after this gets merged.

Would it be safer to split this rename out to a separate PR, get it merged and integrated into the UI, then add the new schemas that have these same names?